swagger/src/main/scala/org/http4s/rho/swagger/package.scala

```diff
@@ -106,10 +71,10 @@ package object swagger {
         Reflector.isPrimitive(t, Set(typeOf[Char], typeOf[Unit]))
 
     def isStream: Boolean =
-      t <:< typeOf[Stream[Task, _]]
+      t <:< typeOf[Stream[_]]
```

Comment:
you removed the fs2 stream import by accident I think.

`def isStream: Boolean = t <: typeOf[Stream[F, _]]` or `Stream[_, _]`

Reply:
True, I noticed that already. I just didn't fix it because I was only able to make it work by hard coding `IO` instead of F. I think this is basically the same problem as with `isEffect` - which also works if I use `IO`. (the code compiles with F, but tests fail)